app/helpers/admin/analytics_helper.rb

```diff
@@ -0,0 +1,11 @@
+module Admin
+  module AnalyticsHelper
+    def track_analytics_data(type, message)
+      {
+        'module' => 'auto-track-event',
+        'track-action' => "alert-#{type}",
+        'track-label' => message,
```

Comment:
Looks ok:
```
$ git grep flash app/controllers/
app/controllers/admin/classification_featurings_controller.rb:        flash[:notice] = "#{@classification_featuring.edition.title} has been featured on #{@classification.name}"
app/controllers/admin/classification_featurings_controller.rb:        flash[:notice] = "#{@classification_featuring.offsite_link.title} has been featured on #{@classification.name}"
app/controllers/admin/classification_featurings_controller.rb:      flash[:notice] = "#{edition.title} has been unfeatured from #{@classification.name}"
app/controllers/admin/classification_featurings_controller.rb:      flash[:notice] = "#{offsite_link.title} has been unfeatured from #{@classification.name}"
app/controllers/admin/edition_unpublishing_controller.rb:      flash.now[:alert] = "The public explanation could not be updated"
app/controllers/admin/edition_workflow_controller.rb:      flash.now[:alert] = @service_object.failure_reason
app/controllers/admin/edition_workflow_controller.rb:      flash.now[:alert] = edition_unwithdrawer.failure_reason
app/controllers/admin/editions_controller.rb:    flash[:notice] = "The document list is being exported"
app/controllers/admin/editions_controller.rb:      flash.now[:alert] = "There are some problems with the document"
app/controllers/admin/editions_controller.rb:      flash.now[:alert] = "There are some problems with the document"
app/controllers/admin/editions_controller.rb:    flash.now[:alert] = "This document has been saved since you opened it"
app/controllers/admin/features_controller.rb:      flash.now[:alert] = "Unable to create feature"
app/controllers/admin/offsite_links_controller.rb:      flash[:notice] = "An offsite link has been created for #{@parent.name}"
app/controllers/admin/offsite_links_controller.rb:    flash[:notice] = "#{@offsite_link.title} has been deleted"
app/controllers/admin/role_appointments_controller.rb:     flash.now[:alert] = "Appointment can not be deleted"
app/controllers/admin/worldwide_organisations_controller.rb:      flash[:notice] = "Main office updated successfully"
```